e2e/src/usecases/HomeFeed.js

```diff
@@ -0,0 +1,72 @@
+import jestExpect from 'expect'
+import { waitForElementId, waitForExpectNotVisible } from '../utils/utils'
+import { sleep } from '../../../src/utils/sleep'
+
+export default HomeFeed = () => {
+  it('should show correct information on tap of feed item', async () => {
+    // Load Wallet Home
+    await waitForElementId('WalletHome')
+    await waitForExpectNotVisible('TransactionList/loading')
+    await waitFor(element(by.id('TransferFeedItem')).atIndex(0))
+      .toBeVisible()
+      .withTimeout(10_000)
+    const feedItems = await element(by.id('TransferFeedItem')).getAttributes()
+
+    // Get address and amount from first item
```

Comment:
Occasionally, amounts on the TX feed and in the TX feed details do not match, e.g., `0.01` & `0.0099`. So we must extract the amounts and assert with jests `toBeCloseTo`.